cypress/pages/members.po.ts

```diff
@@ -1,6 +1,19 @@
-import {Group} from '../utils/member';
-import {wait} from '../utils/wait';
+// Copyright 2020 The Kubermatic Kubernetes Platform contributors.
```

Comment:
Should license headers be there in v2.14?

Reply:
Probably yes, but seems like the [related PR](https://github.com/kubermatic/dashboard/pull/2454) was only merged into master 🤔

Reply:
We can backport it then separately